doc/api/webstreams.md

```diff
@@ -33,6 +33,9 @@ There are three primary types of objects
 * `WritableStream` - Represents a destination for streaming data.
 * `TransformStream` - Represents an algorithm for transforming streaming data.
 
+Additionally, this module includes the utility functions
+`pipeline()`, `finished()`, and `addAbortSignal()`.
```

Comment:
I'm -1 on having this also exported on `stream/web`. Just have the `stream` exports support the additional types.